python/test/unit/io/test_xdmf_meshtags.py

```diff
@@ -13,7 +13,7 @@
 from dolfinx.cpp.mesh import CellType
 from dolfinx.generation import UnitCubeMesh
 from dolfinx.io import XDMFFile
-from dolfinx.mesh import MeshTags, locate_entities_geometrical
+from dolfinx.mesh import MeshTags, locate_entities
```

Comment:
I think all these and below could be locate_entities_boundary as well

Reply:
Can you make a separate PR? This PR is a direct translation so I'd rather not look into other tests at this stage.